package.nls.json

```diff
@@ -101,5 +106,48 @@
   "viewsWelcome.jobLog.noWorkspace": "No workspace folder opened",
   "viewsWelcome.jobLog.scanning": "Scanning for projects...",
   "viewsWelcome.projectExplorer.noWorkspace": "No workspace folder opened",
-  "viewsWelcome.projectExplorer.scanning": "Scanning for projects..."
+  "viewsWelcome.projectExplorer.scanning": "Scanning for projects...",
+  "walkthroughs.gettingStarted.title": "Get Started on a New IBM i Project",
+  "walkthroughs.gettingStarted.description": "Get started on how to set up a new IBM i project",
+  "walkthroughs.createNewProject.title": "Create New Project",
+  "walkthroughs.createNewProject.description": "To get starting on a new IBM i project, create a folder and open it in the workspace.\n[Open Folder](command:workbench.action.files.openFolder)\nFor a workspace folder to be treated as an IBM i project, it must contain an iproj.json file. Navigate to the Project Explorer view and create an iproj.json file for the project.\n[Open Project Explorer](command:projectExplorer.focus)",
+  "walkthroughs.createNewProject.altText": "Create iproj.json action",
+  "walkthroughs.connectToARemoteSystem.title": "Connect to a Remote System",
+  "walkthroughs.connectToARemoteSystem.description": "Connect to an IBM i using the Connections browser.\n[Open Connection Browser](command:vscode-ibmi-projectexplorer.openConnectionBrowser)",
+  "walkthroughs.connectToARemoteSystem.altText": "Open Connection Browser action",
+  "walkthroughs.migrateSourceFromQSYS.title": "Migrate Source From QSYS",
+  "walkthroughs.migrateSourceFromQSYS.description": "To migrate source from QSYS, right-click on any library in the Project Explorer or Object Browser views and use the Migrate Source action.\nThis migration tool will convert all members in the selected source physical files to properly encoded, terminated, and named source files in the IFS and download them to the workspace.\nAs part of this process, this tool can clean up the migrated code to have the source extensions corrected and include/copy directives fixed to use Unix style paths.\n[Learn More](command:vscode-ibmi-projectexplorer.openMigrateSourceDocs)",
```

Comment:
the user just connected to an ibmi and then `right-click on any library in the Project Explorer or Object Browser views`.  Are they going to understand that or does it need to be elaborated a bit?